Description: Use quoted AC_LANG_SOURCE macros when checking for specific
 functionality in configure.ac. This avoids several automake warnings.
Author: Hugh McMaster <hugh.mcmaster@outlook.com>
Last-Update: 2018-04-02

Index: libexif-0.6.21/configure.ac
===================================================================
--- libexif-0.6.21.orig/configure.ac
+++ libexif-0.6.21/configure.ac
@@ -120,23 +120,27 @@ LIBS_orig="$LIBS"
 LIBS="$LIBS -lm"
 AC_MSG_CHECKING([for math functions in libm])
 AC_LINK_IFELSE([
+    AC_LANG_SOURCE([
 	#include <math.h>
 	int main() {
 	  double s = sqrt(0);
 	  double p = pow(s,s);
 	  return (int)p;
 	}
+    ])
 ], [AC_MSG_RESULT(yes)], [
 	AC_MSG_RESULT(no)
 	LIBS="$LIBS_orig"
 	AC_MSG_CHECKING([for math functions without libm])
 	AC_LINK_IFELSE([
+            AC_LANG_SOURCE([
 		#include <math.h>
 		int main() {
 		  double s = sqrt(0);
 		  double p = pow(s,s);
 		  return (int)p;
 		}
+            ])
 	], [
 		AC_MSG_RESULT(yes)
 	],[
